tsl/src/continuous_aggs/invalidation_threshold.c

```diff
@@ -57,6 +57,9 @@
  *                                        |
  *                               invalidation threshold
  *
+ * Transactions that use an isolation level above READ COMMITED don't have
+ * the access to the recent invalidation threshold. Thus such transactions
+ * always create records in the hypertable invalidation log.
  */
```

Comment:
You should point to the original comment which is in `cache_inval_entry_write()` in `tsl/src/continuous_aggs/insert.c`. Also, `COMMITTED` needs 2 `T`.